Xamarin.Forms.Core/Color.cs

```diff
@@ -392,6 +392,16 @@ public static Color FromHsla(double h, double s, double l, double a = 1d)
 			return new Color(h, s, l, a, Mode.Hsl);
 		}
 
+		public static implicit operator System.Drawing.Color(Color color)
+		{
+			return System.Drawing.Color.FromArgb((byte)(color._a * 255), (byte)(color._r * 255), (byte)(color._g * 255), (byte)(color._b * 255));
```

Comment:
I'm quite sure this will fail, or get you unexpected results
```
   var sdcolor = (System.Drawing.Color)Xamarin.Forms.Color.Default;
```

Reply:
Color.Default has its fields (r, g, b and a) initialized to `-1`. converting to a S.D.Color results of a S.D.Color(1,1,1,1);

That's definitely not what you want.

if you want to match X.F.Color.Default to something, it should converts to S.D.Color.Empty (and back)